adobe/spectrum-web-components
# build: prevent "cem" from churning the package.json entry for the repo

## Description
Use the latest version of CEM/Analyzer so that we can stop churning on changes in the root level `package.json` even though there are no custom elements there.

## How has this been tested?
    1. Pull the branch.
    2. `yarn` to get the latest deps
    3. `yarn storybook` to run Storybook
    4. See that `package.json` isn't updated
    1. Repeat the above with `yarn docs:start` to see that the same happens with the local docs site.

## Types of changes

## Checklist
